部分优化

BozaiIZS opened the pull request:
删除slidewindow，使用Preprocessor替代
删除了老版的ssa和fft
在omni上使用和xception同样的预划分数据方法

BozaiIZS committed:
测试

BozaiIZS committed:
Delete 1.py

BozaiIZS committed:
修改训练ominiscalecnn，使用预划分数据验证

BozaiIZS committed:
删除slidewindow，使用preprocessor，修改部分格式

zhaosiyuan1098 reviewed (changes requested):
1.误删了model_compare函数
2.在main.py中，omniscale CNN不能将fft结果和原始信号拼接起来训练，只用fft就好
3.evaluate函数可删去，因为在train中已经集成了相关的验证

BozaiIZS committed:
2024.5.26优化

1.取消删除注释掉的model_compare函数
2.在main.py中，omniscale CNN只用fft处理的数据
3.删除omni和xcep中的evaluate函数
